### DISCUSSION TITLE
Reorganization of files

### DISCUSSION THREAD

**Discussion by FormulaCarbon** [Coderwe1/GalaxyOS#15] [Ideas]
Hi!
I just got accepted and went to check out the github. noticed it is messy and doesnt follow regular website tree structure. I dont now if the repl is like this but if it is it should be changed immediatly.

proper file structure

*
|-- static
|    |-- css
|         |--ALL CSS FILES
|-- templates
|    |-- ALL HTML FILES
|-- dynamic
     |-- ALL JS AND/OR PYTHON FILES

**Comment by thatfrozenfrog** [comment]
Indeed, thats a good idea

**Reply by Coderwe1** [reply]
yes

**Comment by Coderwe1** [comment]
@FormulaCarbon can you format the github page(I dont know the format)

**Comment by Coderwe1** [comment]
Also we will need to change there file paths for all the links
